CMakeLists.txt

```diff
@@ -25,5 +29,19 @@ if(BUILD_TESTING)
 endif()
 
 add_subdirectory(src/beman/exemplar)
-
 add_subdirectory(examples)
+
```

Comment:
Shouldn't all the library metadata (i.e., `exemplar-config.cmake`, etc.) go in the `src/beman/exemplar` subdir?